shared/constants/swaps.ts

```diff
@@ -24,7 +24,8 @@ export const MAX_ALLOWED_SLIPPAGE = 15;
 
 // An address that the metaswap-api recognizes as the default token for the current network,
 // in place of the token address that ERC-20 tokens have
-const DEFAULT_TOKEN_ADDRESS = '0x0000000000000000000000000000000000000000';
+export const DEFAULT_TOKEN_ADDRESS =
```

Comment:
Minor: there's also a shared constant we can use `BURN_ADDRESS` but I've been using `zeroAddress()` . Thoughts on making our usage within bridging consistent?

Reply:
I prefer using `zeroAddress()` over `BURN_ADDRESS`, just because we aren't looking to burn anything. We just happen to be using 0x0...0 as the native token address. 

And yes agreed with keeping things consistent within Bridge.

Reply:
Oh `export const BURN_ADDRESS = zeroAddress();`, literally the same thing

Reply:
I'm good with `BURN_ADDRESS` then

Reply:
Yes agreed with keeping it consistent within Bridge

Reply:
We can continue to use `zeroAddress()`, I'll update my code

Reply:
Done in [0ddc39c](https://github.com/MetaMask/metamask-extension/pull/27262/commits/0ddc39cb27a4bae3077fab5f572ced8c16be102a)